File: src/server/Shared/Shared.DTOs/Enums/AttendanceStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FluentPOS.Shared.DTOs.Enums
{
    public enum AttendanceStatus
    {
        None = 0,
        Present = 1,
        Absent = 2,
        Leave = 3,
        Holiday = 4,
        Off = 5
    }
}

File: src/client/src/app/core/enums/AttendanceStatus.ts
export enum AttendanceStatus {
    None = 0,
    Present = 1,
    Absent = 2,
    Leave = 3,
    Holiday = 4,
    Off = 5
}
export const AttendanceStatusMapping: Record<AttendanceStatus, string> = {
    [AttendanceStatus.None]: "-",
    [AttendanceStatus.Present]: "Present",
    [AttendanceStatus.Absent]: "Absent",
    [AttendanceStatus.Leave]: "Leave",
    [AttendanceStatus.Holiday]: "Holiday",
    [AttendanceStatus.Off]: "Off Day"
};
